drivers/include/net/netdev.h

```diff
@@ -271,6 +272,19 @@ typedef struct netdev netdev_t;
  */
 typedef void (*netdev_event_cb_t)(netdev_t *dev, netdev_event_t event);
 
+/**
+ * @brief   Driver types for netdev.
+ *          To be used for @see netdev_register and to match EUI providers.
+ * @{
+ */
+typedef enum {
+    NETDEV_ANY = 0,         /**< Will match any device type */
+    NETDEV_AT86RF215,
+    NETDEV_AT86RF2XX,
+    NETDEV_DOSE,
+} netdev_type_t;
```

Comment:
Maybe add a comment here that this `enum` may be extended:

```suggestion
    /* add more if needed */
} netdev_type_t;
```

Reply:
added